### DISCUSSION TITLE
GDG Event Serverpod

### DISCUSSION THREAD

**Discussion by II11II** [serverpod/serverpod#478] [General]
Where can I find the recorded video from this event? 
@vlidholt 
https://twitter.com/GdgCloudSthlm/status/1593182263721840642?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1593182263721840642%7Ctwgr%5E87d5317729fa12442497e80366cfb5e4fbdff3d0%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fpublish.twitter.com%2F%3Fquery%3Dhttps3A2F2Ftwitter.com2FGdgCloudSthlm2Fstatus2F1593182263721840642widget%3DTweet

**Comment by tusharbhambere** [comment]
I don't think that event video recording Publicly Available :(
But you can watch this lates Serverpod Video
https://youtu.be/D20VmP6n6SM

**Comment by vlidholt** [comment]
Sadly I had the worst man-cold, so I couldn't do the presentation. Fever and no voice. :(

However, I'm doing another GDG event on Saturday (Mons in Belgium), and there are many others coming soon too!

**Reply by AndryHTC** [reply]
@vlidholt will there be any live link? 😄

**Reply by vlidholt** [reply]
I'm not sure, I'll keep you posted! I think it'll be recorded though.

**Reply by vlidholt** [reply]
It will be streamed live on Youtube. Tomorrow, Saturday at 1.30pm CET. Here is the link: https://www.youtube.com/watch?v=XgpZ-U5lHy4
